### PR TITLE
set base site to localhost when scrubbing db

### PR DESCRIPTION
**Pull request by shapiromatron**
in scrub db command, set Site to localhost to prevent admin urls going to production site

### CODE REVIEW

File: hawc/apps/myuser/management/commands/scrub_db.py [hunk 1]
```diff
@@ -33,6 +34,13 @@ def handle(self, *args, **options):
             if input("".join(message)) != "yes":
                 raise CommandError("Scrubbing user data cancelled.")
 
+        self.update_site()
+        self.update_users()
+
+    def update_site(self):
+        Site.objects.update(domain="127.0.0.1:8001", name="localhost")
```
Comment (casey1173) [inline hawc/apps/myuser/management/commands/scrub_db.py:41]: should this be 8000 instead of 8001? I think it's the default case

**Review by casey1173 — COMMENTED** [review]
Looks good and works well, just one question about the url

File: hawc/apps/myuser/management/commands/scrub_db.py (hunk 1 above)
Comment (shapiromatron) [reply hawc/apps/myuser/management/commands/scrub_db.py:41]: oh, yeah, wow good catch!

Comment (casey1173) [reply hawc/apps/myuser/management/commands/scrub_db.py]: okay, changed it to 8000! all good now

**Review by casey1173 — APPROVED** [review]

**Pull request merged**